src/pages/profile-page/profile-page.tsx

```diff
@@ -0,0 +1,105 @@
+import { useUserClient } from '../../hooks/api/use-user-client';
+import { Deck } from '../../models/deck';
+import { paths } from '../../routing/paths';
+import { userInfoStateAsync } from '../../state/auth-jwt';
+import { userDecksSortedState, userDecksState } from '../../state/user-decks';
+import './profile-page.scss';
+
+export const ProfilePage = () => {
+  return (
+    <React.Suspense fallback={<LoadingPage />}>
+      <AsyncProfilePage />
+    </React.Suspense>
+  );
+};
+
+const AsyncProfilePage = () => {
+  const navigate = useNavigate();
+  const userClient = useUserClient();
+  const decksClient = useDecksClient();
+
+  const setUserDecks = useSetRecoilState(userDecksState);
+  const sortedDecks = useRecoilValue(userDecksSortedState);
+  const userData = useRecoilValue(userInfoStateAsync);
+
+  const pfpUrl = userClient.getProfilePictureUrl(userData?.email ?? '');
+  const { privateDecks, publicDecks } = _reduceDecksByAccess();
+
+  // fetch flashcard decks on load
+  useEffect(() => {
+    fetchDecksAndRefresh();
+  }, []);
+
+  return (
+    <div className="pg-profile-page">
+      <div className="profile-page-header">
+        <PageHeader label="my profile" goBackLabel="go back" onGoBackClick={() => navigate(-1)} />
+      </div>
+      <div className="user-details">
+        <div className="profile-picture-container">
+          <img className="profile-picture" src={pfpUrl} loading="lazy" />
+        </div>
+        <span className="username">{`@${userData?.username}`}</span>
+        <span className="full-name">{userData?.email.toLowerCase()}</span>
+        <span className="date-joined">member since 2022</span>
+      </div>
+      <BubbleDivider
+        className="decks-divider"
+        variantType="drop-down-reveal"
+        label={`private decks (${privateDecks.length})`}
+      >
+        {privateDecks.length > 0 && (
+          <div className="decks-container">{getDeckCovers(privateDecks)}</div>
+        )}
+      </BubbleDivider>
+      <BubbleDivider
+        className="decks-divider"
+        variantType="drop-down-reveal"
+        label={`public decks (${publicDecks.length})`}
+      >
+        {publicDecks.length > 0 && (
+          <div className="decks-container">{getDeckCovers(publicDecks)}</div>
+        )}
+      </BubbleDivider>
+    </div>
+  );
+
+  function getDeckCovers(decks: Deck[]) {
+    return decks.map((deck) => {
+      const deckId = deck.metaData.id;
+      return (
+        <DeckCover
+          key={deckId}
+          deck={deck}
+          onStudyClick={() => navigate(`${paths.study}/${deckId}`)}
+          onViewClick={() => navigate(`${paths.deck}/${deckId}`)}
+        />
+      );
+    });
+  }
+
+  function _reduceDecksByAccess() {
+    return sortedDecks.reduce(
+      (result: Record<'privateDecks' | 'publicDecks', Deck[]>, deck: Deck) => {
+        const accessLevel = deck.metaData.access;
+        const key = accessLevel === 'Private' ? 'privateDecks' : 'publicDecks';
```

Comment:
Wait, do we already have the access level stuff set up??

Reply:
all decks are Public, but we have no endpoints to alter it atm.

Reply:
oh, haha. I was thinking we were slacking on the front-front editor, lol. But we can just add a dropdown/radio button once they get that endpoint up in the deck editor

Reply:
i'm actually completely wrong `POST /Decks/:id` can modify the access level
so yeah, this is something we just need to add to the metadata dropdown

Reply:
Sounds good. I'll make a PR for that